Source/TractUI.cpp

```diff
@@ -196,12 +295,11 @@ void TractUI::moveTo(Graphics &g, t_tractProps *props, Path &p, double i, double
 {
 	double angle = this->angleOffset + i * this->angleScale * M_PI / (props->lipStart - 1);
 	
-	// TODO: Figure out wobble
-//	double time = 0.0;
-//	double wobble = (Tract.maxAmplitude[Tract.n-1]+Tract.noseMaxAmplitude[Tract.noseLength-1]);
-//	wobble *= 0.03 sin(2 * i - 50 * time) * i / (double) props->n;
-//	angle += wobble;
-	double wobble = 0.0;
+    time_t t = time(NULL);
+    struct tm * tm = localtime(&t);
+	double wobble = props->maxAmplitude[props->n-1]+props->noseMaxAmplitude[props->noseLength-1];
```

Comment:
Since this logic is duplicated in `lineTo`, I think it makes sense to consolidate the two functions into one, with a branch depending on whether we're starting a new path or else drawing a line.

Reply:
I ended up extracting the calculation of the r and angle coordinates into a separate function and leaving the two functions before seeing this comment. I think what I implemented also works to simplify but I can do what you suggested if you think that's better